tests/integration/test_npm.py

```diff
@@ -0,0 +1,66 @@
+import logging
+from pathlib import Path
+
+import pytest
+
+from . import utils
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize(
+    "test_params,lockfile_version",
+    [
+        # Test case checks loading npm dependencies in SBOM format for source repo
+        # Plus fetching the dependencies
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/npm-cachi2-smoketest.git",
+                ref="bae083d57dc265a899b59859b769e88eb8319404",
```

Comment:
It seems that this commit does not exist, and is causing CI to fail.

Reply:
Nice catch. Thank you! Fixed